resources/views/pages-legacy/leaderboardList.blade.php

```diff
@@ -26,7 +26,7 @@
 $gameData = getGameData($gameID);
 $codeNotes = [];
 if ($permissions >= Permissions::JuniorDeveloper) {
-    getCodeNotes($gameID, $codeNotes);
+    $codeNotes = getCodeNotesData($gameID);
 }
```

Comment:
codeNotes does not appear to be used on this page. these four lines (27-30) could be eliminated.